backend/configs/usr/lib/cgi-bin/factory_reset.py

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+from cgi import FieldStorage
+
+
+def _error(msg=""):
```

Ruff (0.8.2)

Missing return type annotation for private function `_error`

Add return type annotation: `None`

(ANN202)